### DISCUSSION TITLE
Inquiry about Chat Storage File in GPTMobile v0.5.3 for Backup

### DISCUSSION THREAD

**Discussion by X4n71uM** [Taewan-P/gpt_mobile#143] [Q&A] [answered]
Hi Taewan Park,

I hope this message finds you well. I am currently using GPTMobile version 0.5.3 and I am interested in creating a backup of my chat history. Could you please guide me on which file or directory the chat data is stored in within the app?

Your assistance would be greatly appreciated.

Thank you for your time and support.

Best regards,
Xantium

**Comment by Taewan-P** [comment, marked as answer]
Hello,

Currently, the chat history is not supported.
For chat history backups, the only way right now is to root your phone, and gain access to `/data/data/dev.chungjungsoo.gptmobile` and extract the sqlite db file.

If you installed an debug APK, you do not require root since it can be accessed via Android Studio (File Explorer) or via ADB.

**Comment by X4n71uM** [comment]
Thx, bro 🤓
